lib/Discord/events.ex

```diff
@@ -5,7 +5,7 @@ defmodule Alchemy.Discord.Events do
   # This module is then used by EventStage.Cache
```

Comment:
Just remove the lines instead of commenting them out. We're using git so we don't have to comment things out :P